source/documentation/other-topics/custom-domain-cert.md

```diff
@@ -48,16 +55,16 @@ material will be stored, in your namespace.
        - domains:
          - '<hostname>'
          dns01:
-           provider: <provider>
+           provider: route53-cloud-platform
    ```
 
-3. Make sure the certificate has been issued correctly, by checking its `Status`:
+2. Make sure the certificate has been issued correctly, by checking its `Status`:
 
    ```
    $ kubectl describe certificate <my-cert>
    ```
 
-4. You will need to update your `Ingress` spec to include the new hostname.
+3. You will need to update your `Ingress` spec to include the new hostname.
 
    **Once your host is defined here, the cluster will take control of the DNS record and automatically adjust to point to the cluster.**
```

Comment:
What does this mean for the user? What will they be able to do, once this has happened? i.e. how can they test that it's working, or not. How long will it normally take for the cluster to do its thing? i.e. when should they ask for help?

Reply:
Also, I think it would be clearer here to move the explanatory text after the code sample.
i.e. `3. You will need to update your ingress ... code sample ... explanatory text`

Reply:
I considered removing this note altogether. It was written with migrating teams in mind (because they would have had a record pointing to their template-deploy setup). I can see how it's not clear at all.

Perhaps I should re-write this in the section about setting up NS records? That is when the cluster truly assumes control of the zone. What do you think?